<?xml version="1.0"?>
<launch>
  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf" output="screen">
    <rosparam command="load" file="$(find metu_navigation)/config/ekf_localization_node/ekf_2d.yaml" />
    <param name="output_frame" value="odom" />
    <param name="base_footprint_frame" value="base_link" />
    <param name="freq" value="30.0" />
    <param name="sensor_timeout" value="1.0" />
    <param name="odom_used" value="true" />
    <param name="imu_used" value="true" />
    
    <param name="publish_tf" value="true" />
    <param name="debug" value="false" />
    <param name="gps_used" value="false" />
    <param name="vo_used" value="true" />

    <!--remap from="odom" to="/controllers/diff_drive/odom" /-->
    <remap from="odom" to="/zed2/odom" />
    <!--remap from="imu_data" to="/zed2/imu/data" /-->
    <!--remap from="gps" to="/odom" /-->
    <!--remap from="vo" to="/rtabmap/odom"/-->

  </node>

  <!--node pkg="metu_artag" type="tf_base_link_broadcaster_node.py" name="tf_base_link_broadcaster_node" output="screen" /-->
</launch>
